enterprise/fulltext-addon/src/main/java/org/neo4j/kernel/api/impl/fulltext/integrations/bloom/BloomProcedure.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.kernel.api.impl.fulltext.integrations.bloom;
+
+import java.util.ArrayList;
+
+import org.neo4j.collection.RawIterator;
+import org.neo4j.collection.primitive.PrimitiveLongIterator;
+import org.neo4j.kernel.api.exceptions.ProcedureException;
+import org.neo4j.kernel.api.exceptions.Status;
+import org.neo4j.kernel.api.impl.fulltext.FulltextProvider;
+import org.neo4j.kernel.api.impl.fulltext.ReadOnlyFulltext;
+import org.neo4j.kernel.api.proc.CallableProcedure;
+import org.neo4j.kernel.api.proc.Context;
+import org.neo4j.kernel.api.proc.Neo4jTypes;
+import org.neo4j.kernel.api.proc.QualifiedName;
+
+import static org.neo4j.kernel.api.proc.ProcedureSignature.procedureSignature;
+
+/**
+ * Procedures for querying the bloom fulltext addon.
+ */
+public class BloomProcedure extends CallableProcedure.BasicProcedure
+{
+    private static final String OUTPUT_NAME = "entityid";
+    private static final String PROCEDURE_NAME = "bloomFulltext";
+    private static final String[] PROCEDURE_NAMESPACE = {"db", "fulltext"};
```

Comment:
i think Michael/Antons  comment about official API support for procedures is  still valid, would be nice if we use only officially supported/promoted way